Doc/library/sys.rst

```diff
@@ -1253,7 +1253,7 @@ always available.
 
    The trace function is invoked (with *event* set to ``'call'``) whenever a new
    local scope is entered; it should return a reference to a local trace
-   function to be used that scope, or ``None`` if the scope shouldn't be traced.
+   function to be used for the new scope, or ``None`` if the scope shouldn't be traced.
```

Comment:
The suggested changes causes this line to go above the maximum of 80 for chars per line (currently at 87). This can be easily fixed by splitting the last word into the next line:
```suggestion
   function to be used for the new scope, or ``None`` if the scope shouldn't be
   traced.
```

Reply:
Done! I just commited your suggestion.